src/tctrack/tempest_extremes/tempest_extremes.py

```diff
@@ -968,6 +968,9 @@ def _parse_tracks_gfdl(self, file_path):
         tracks = {}
         current_track_id = 0  # Initialize track ID
 
+        # Get variable names from in_fmt
+        var_names = self.stitch_nodes_parameters.in_fmt or []
```

Comment:
I think the below may be 'safer' as `_parse_gfdl_line_to_point` expects a list of variable names or `None` as an input. Whilst `if([])` returns `False` in Python there is perhaps potential for confusion later, whilst doing this would propogate the list of names or `None` directly as set in `self.stitch_nodes_parameters.in_fmt`

```suggestion
        var_names = self.stitch_nodes_parameters.in_fmt
```

Reply:
Good point, I just got confused myself when I looked at what it was currently doing.